instrumentation/http_instrumentation.go

```diff
@@ -1,13 +1,18 @@
-package server
+package instrumentation
```

Comment:
Just a small nit, but we do have separate instrumentation code than this like `logger.NewLogger`. So this might lead to some confusion for contributors  (for eg which to use `instrumentation.Logger` or `logger.NewLogger`?).

Maybe we can rename this package to `http_instrumention` instead?

Reply:
It shouldn't lead to confusion: `instrumentation.Logger` has documentation explaining it's a logger middleware. We can guess, because of the lack of documentation, that `logger.NewLogger` creates a logger because of its name and the package where it lives.